actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -29,7 +29,16 @@ def channel_class_names
           Pathname.new(channel_path).basename.to_s.split('.').first.camelize
         end
       end
+
+      # Returns constant of subscription adapter specified in config/cable.yml
+      # If the adapter cannot be found, this will default to the Redis adapter
+      def subscription_adapter
+        # Defaults to redis if no adapter is set
+        adapter = cable.fetch('adapter') { 'redis' }
+        adapter = adapter.camelize
+        adapter = "PostgreSQL" if adapter == "Postgresql"
```

Comment:
We could just call the adapter `Postgres` and spare this munging (otherwise to do this right, we'd need a more advanced registry, which is too much).